apps/server/src/adapterSetup.ts

```diff
@@ -1,35 +1,20 @@
-import { adapterMapObject as testAdapterMapObject } from './test-adapter'
-import { MxAdapter } from './adapters/mx'
-import { SophtronAdapter } from './adapters/sophtron'
+import { adapterMapObject as testAdapterMapObject } from "./test-adapter";
+import { SophtronAdapter } from "./adapters/sophtron";
 
-import { mxIntGetVC, mxProdGetVC } from './services/vcAggregators/mxVc'
-import getSophtronVc from './services/vcAggregators/sophtronVc'
-
-const mxAdapterMapObject = {
-  mx: {
-    testInstitutionAdapterName: 'mx_int',
-    vcAdapter: mxProdGetVC,
-    widgetAdapter: new MxAdapter(false)
-  },
-  mx_int: {
-    vcAdapter: mxIntGetVC,
-    widgetAdapter: new MxAdapter(true)
-  }
-}
+import getSophtronVc from "./services/vcAggregators/sophtronVc";
 
 const sophtronAdapterMapObject = {
   sophtron: {
     vcAdapter: getSophtronVc,
-    widgetAdapter: new SophtronAdapter()
-  }
-}
+    widgetAdapter: new SophtronAdapter(),
+  },
+};
 
 // This is where you add adapters
 export const adapterMap = {
-  ...mxAdapterMapObject,
```

Comment:
Removing this would be a breaking change. We need to use the package and put it here until we do a version 1.0.0 that requires you to import your own adapters

Reply:
I don't understand. If I shouldn't remove this, then all the other code I removed needs to go back in as well. Without the other mx-related code, this line that you commented on won't work either.

Reply:
Oh, I understand now. You're saying import the published MX package, and use that until we're ready for the packages to be used.

Reply:
Yes. Until we do a 1.0.0 we need to import the published packages and use them in here. We can do a breaking change in 1.0.0 where we expect people to import their own adapters.

Reply:
Okay, @wesrisenmay-mx This is done.